http://sourceforge.net/tracker/?func=detail&aid=3540867&group_id=309786&atid=1304474

warning: implicit declaration of function ‘free’ and 'realloc' [-Wimplicit-function-declaration]

--- askpass.c
+++ askpass.c
@@ -23,6 +23,7 @@
 #include <sys/resources.h>
 #endif
 
+#include <stdlib.h> /* free */
 #include <glib.h>
 
 #ifdef HAVE_GNOME_KEYRING
--- secureentry.c
+++ secureentry.c
@@ -18,6 +18,7 @@
 #include "config.h"
 
 #include <unistd.h>
+#include <stdlib.h> /* free and realloc */
 #include <string.h>
 #ifdef HAVE_GNOME_KEYRING
 #include <gnome-keyring-memory.h>
